From mingo@elte.hu  Fri Jun  6 14:31:01 2008
From: Ingo Molnar <mingo@elte.hu>
Date: Mon, 2 Jun 2008 21:21:03 +0200
Subject: USB: fix build bug in USB_ISIGHTFW
To: Matthew Garrett <mjg59@srcf.ucam.org>
Cc: linux-usb@vger.kernel.org, linux-kernel@vger.kernel.org, gregkh@suse.de, bersace03@laposte.net, johannes@sipsolutions.de
Message-ID: <20080602192103.GA31589@elte.hu>
Content-Disposition: inline


From: Ingo Molnar <mingo@elte.hu>

USB: fix build bug in USB_ISIGHTFW

-tip tree testing found this build bug:

  drivers/built-in.o: In function `isight_firmware_load':
  isight_firmware.c:(.text+0x1ade08): undefined reference to `request_firmware'
  isight_firmware.c:(.text+0x1adf9c): undefined reference to `release_firmware'

select FW_LOADER in USB_ISIGHTFW.


From: Ingo Molnar <mingo@elte.hu>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/misc/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/usb/misc/Kconfig
+++ b/drivers/usb/misc/Kconfig
@@ -272,6 +272,7 @@ config USB_TEST
 config USB_ISIGHTFW
 	tristate "iSight firmware loading support"
 	depends on USB
+	select FW_LOADER
 	help
 	  This driver loads firmware for USB Apple iSight cameras, allowing
 	  them to be driven by the USB video class driver available at
